upload_run.py

```diff
@@ -69,14 +36,119 @@ def __call__(self, parser, namespace, values, option_string=None):
                                   'The list of runs is generated at start time '
                                   '(Runs added to directory mid upload will not be uploaded).')
 
+# Optional arguments for overriding config file settings
+# Explanation:
+#   nargs='?', const=True, default=False,
+#       Allows zero or one parameters
+#       If the argument is not given:                      the value will be False           (indicates load from file)
+#       If the argument is given, and no parameter given:  the value will be True            (prompt user for input)
+#       If the argument is given, and parameter is given:  the value will be the parameter   (used to override config)
+argument_parser.add_argument('-ci', '--config_client_id', action='store', nargs='?', const=True, default=False,
+                             help='Override "client_id" in config file. '
+                                  'Can be used without a parameter to prompt for input.')
```

Comment:
Rather than the
> Override "client_id" in config file. Can be used without a parameter to prompt for input.

message, can you say what the options are for?  You could put something at the bottom of the message that you can use all `-c*` options without a parameter to prompt for input.

Reply:
I added the `-c*` message to the help epilog, let me know if you think the wording should be changed at all.